### PR TITLE
[#29] Create .travis.yml file for Travis CI

### LINKED ISSUE #29: Create .travis.yml file for Travis CI
Now that NNA has a Travis CI instance lets configure the build for it in the `.travis.yml` file.

### PR DESCRIPTION
**Pull request by zero45**
Make sure you have checked all steps below.

### GitHub Issue
Fixes #29 

### Checklist:

- [ ] This pull request updates the documentation
- [X] This pull request changes the code
- [X] Title of the PR is of format (example) : `[#25] Add Pull Request Template`, where `[#25] is the GitHub issue number`

### What is the purpose of this pull request?
To enable Travis CI builds.

### How was this change validated?
Ensured build ran on https://travis-ci.com/paypal/NNAnalytics/builds/75449736
Ensured that it builds all supported Hadoop versions and checks findbugs, license, and tests

### Commit Guidelines
- [X] My commits all reference GH issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  fast_finish: true
+  include:
+    - env: VERSION='-PhadoopVersion=2.4.0'
+    - env: VERSION='-PhadoopVersion=2.6.1'
+    - env: VERSION='-PhadoopVersion=2.7.0'
+    - env: VERSION='-PhadoopVersion=2.8.0'
+    - env: VERSION='-PhadoopVersion=2.9.0'
+    - env: VERSION='-PhadoopVersion=3.0.0'
+script:
+  - ./gradlew clean build licenseMain licenseTest findbugsMain findbugsTest buildRpm $VERSION
```
Comment (jhsenjaliya) [inline .travis.yml]: should we add checkStyle here ?

Comment (r39132) [reply .travis.yml]: Yes... let’s definitely add that

Comment (zero45) [reply .travis.yml]: We will as a part of wrapping up #23

Comment (zero45) [reply .travis.yml]: Oh sorry -- it's actually already there anyway guys. Take a look at the job log here: https://travis-ci.com/paypal/NNAnalytics/jobs/127776725

You can see its already running checkstyle as part of the `build` operation.
I confirmed this locally as well. I forgot it does that.

Comment (jhsenjaliya) [reply .travis.yml]: confirming that spotlessJava runs as part of `gradle build`, and also fails if there is violatioin. although it also runs the checkstyle for me, probably should remove it from `gradle apply plugin`.
I think we should merge this, and take care of larger checkstyle issues on issue #21 .
+1, LGTM

Comment (zero45) [reply .travis.yml]: Okay we are in agreement on proceeding AFAICS. I will commit shortly then. Thank you!

**Pull request merged**